DigitalLearningSolutions.Web.Tests/Controllers/Login/LoginControllerTests.cs

```diff
@@ -342,5 +342,57 @@ public void Multiple_approved_accounts_at_same_centre_should_log_in()
             result.Should().BeRedirectToActionResult()
                 .WithControllerName("Home").WithActionName("Index");
         }
+
+        [Test]
+        public void
+            When_user_has_accounts_with_inactive_centres_only_check_for_centres_that_are_active()
+        {
+            // Given
+            var expectedAdminUser = UserTestHelper.GetDefaultAdminUser(centreId: 1, centreName: "Centre 1", centreActive: false);
+            var expectedDelegateUsers = new List<DelegateUser>
+            {
+                UserTestHelper.GetDefaultDelegateUser(centreId: 1, centreName: "Centre 1", centreActive: false),
+                UserTestHelper.GetDefaultDelegateUser(centreId: 2, centreName: "Centre 2", centreActive: true)
+            };
+            A.CallTo(() => userService.GetUsersByUsername(A<string>._))
+                .Returns((expectedAdminUser, expectedDelegateUsers));
+            A.CallTo(() => loginService.VerifyUsers(A<string>._, A<AdminUser>._, A<List<DelegateUser>>._))
+                .Returns((expectedAdminUser, expectedDelegateUsers));
+            var expectedApprovedDelegateUsers = expectedDelegateUsers.Where(du => du.CentreActive).ToList();
+
+            // When
+            controller.Index(LoginTestHelper.GetDefaultLoginViewModel());
+
+            // Then
+            A.CallTo(() => userService.GetUserCentres(
+                    null,
+                    A<List<DelegateUser>>.That.Matches(list => list.SequenceEqual(expectedApprovedDelegateUsers))))
+                .MustHaveHappened();
+        }
+
+        [Test]
+        public void
+            When_user_has_verified_accounts_only_at_inactive_centres_then_redirect_to_centre_inactive_page()
+        {
+            // Given
+            var expectedAdminUser = UserTestHelper.GetDefaultAdminUser(centreId: 1, centreName: "Centre 1", centreActive: false);
+            var expectedDelegateUsers = new List<DelegateUser>
+            {
+                UserTestHelper.GetDefaultDelegateUser(centreId: 1, centreName: "Centre 1", centreActive: false)
+            };
+            A.CallTo(() => userService.GetUsersByUsername(A<string>._))
+                .Returns((expectedAdminUser, expectedDelegateUsers));
+            A.CallTo(() => loginService.VerifyUsers(A<string>._, A<AdminUser>._, A<List<DelegateUser>>._))
+                .Returns((expectedAdminUser, expectedDelegateUsers));
+            A.CallTo(() => userService.GetUserCentres(expectedAdminUser, expectedDelegateUsers))
+                .Returns(new List<CentreUserDetails>());
+
+            // When
+            var result =
+                controller.Index(LoginTestHelper.GetDefaultLoginViewModel());
```

Comment:
Super minor, but should this be on a new line? (it doesn't seem like a particularly long line of code)